recipes/datacube-compute/recipe.yaml

```diff
@@ -0,0 +1,52 @@
+context:
+  name: datacube-compute
+  version: 0.0.6
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/auspatious/datacube-compute/archive/${{ version }}.tar.gz
+  sha256: 4706e086e4fb5049c88cfd28ea2dfda2f40c56146834e09fb59e8956d5be801a
+
+build:
+  number: 0
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }} ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: win
```

Comment:
Haven't quite figured out the syntax to set an environment variable after looking at https://github.com/conda/ceps/blob/main/cep-0014.md#script-section (see attempt at 2f2c4dfd4826d574fe98c0e9444563945d1e45d8), so skipping Windows builds for now.